src/operator/rnn-inl.h

```diff
@@ -1362,7 +1427,9 @@ class RNNOp {
   std::vector<cudnnTensorDescriptor_t> x_desc_vec_, y_desc_vec_, dx_desc_vec_, dy_desc_vec_;
   #if USE_CUDNN_LSTM_PROJ
   cudnnRNNDataDescriptor_t x_data_desc_, y_data_desc_, dx_data_desc_, dy_data_desc_;
+  DType padding_fill_ = 0;
   #endif
+  std::vector<IType> sequence_length_array_cpu;
```

Comment:
instead of allocating in the operator, try use the shared temp resource for this.

request: https://github.com/apache/incubator-mxnet/blob/master/src/operator/tensor/indexing_op.cc#L530-L533
usage: https://github.com/apache/incubator-mxnet/blob/master/src/operator/tensor/indexing_op.cc#L365
get_host_space: https://github.com/apache/incubator-mxnet/blob/master/include/mxnet/resource.h#L143

Reply:
Done.